tests/run-pass/intrinsics-math.rs

```diff
@@ -85,4 +85,6 @@ pub fn main() {
     assert_approx_eq!(1.0f32.tan(), 1.557408f32);
     assert_approx_eq!(1.0f64.tan(), 1.557408f64);
 
+    assert_approx_eq!(0.65f32.ldexp(3.0f32 = 5.2f32);
```

Comment:
Looks like unmatched parenthesis? Please run `./miri test` locally before opening a PR. :)

Reply:
Yeah sorry, I cannot build miri right now for some reason, there are a lot of errors if I try to build it with rust/master or with the current nightly I have. I'll fix it

Reply:
`rustup-toolchain-install-master $(cat rust-version) -c rust-src` will install a version of Rust that works. Latest nightly will also do it as of now, that's what I am using.

Submitting untested code for review tends to waste a lot of time, so please fix your setup.

Reply:
Thank you! Now I can run the tests and they in fact do not pass because I haven't added `ldexp` to the `f64` and `f32` types. Should I do that first?

Reply:
Not sure what you mean by "add to these types"?

You should call that method the same way https://github.com/rust-lang/rust/blob/0f11354a9c1bf0c5ac250c7fa2bafc289a662f42/src/libcore/tests/num/flt2dec/mod.rs#L82-L91 does.